docker-compose.yml

```diff
@@ -1,75 +1,60 @@
 wordpress:
-  build: .
-  privileged: true
-  env_file: _env
-  environment:
-    - CONTAINERBUDDY=file:///etc/containerbuddy.json
   restart: always
   mem_limit: 1g
-  labels:
-    - triton.cns.services=wordpress
+  env_file: _env
   ports:
     - 80
+  labels:
+    - triton.cns.services=wordpress
 
 nfs:
   image: autopilotpattern/nfsserver
-  privileged: true
-  env_file: _env
-  mem_limit: 256m
   restart: always
-  labels:
-    - triton.cns.services=nfs
+  mem_limit: 256m
+  env_file: _env
   expose:
     - 111
     - 1892
     - 2049
+  labels:
+    - triton.cns.services=nfs
 
 mysql:
   image: autopilotpattern/mysql:latest
+  restart: always
   mem_limit: 4g
   env_file: _env
-  restart: always
-  labels:
-    - triton.cns.services=mysql
   expose:
     - 3306
+  labels:
+    - triton.cns.services=mysql
 
 memcached:
   image: autopilotpattern/memcached:latest
+  restart: always
   mem_limit: 512m
   env_file: _env
-  environment:
-    - CONTAINERBUDDY=file:///etc/containerbuddy.json
```

Comment:
Dockerfile for the memcached image has already been updated:

```
ENV CONTAINERPILOT_VER 2.0.1
RUN export CONTAINERPILOT_CHECKSUM=a4dd6bc001c82210b5c33ec2aa82d7ce83245154 \
    && curl -Lso /tmp/containerpilot.tar.gz \
        "https://github.com/joyent/containerpilot/releases/download/${CONTAINERPILOT_VER}/containerpilot-${CONTAINERPILOT_VER}.tar.gz" \
    && echo "${CONTAINERPILOT_CHECKSUM}  /tmp/containerpilot.tar.gz" | sha1sum -c \
    && tar zxf /tmp/containerpilot.tar.gz -C /usr/local/bin \
    && rm /tmp/containerpilot.tar.gz

# Add ContainerPilot configuration
COPY etc/containerpilot.json /etc/containerpilot.json
ENV CONTAINERPILOT file:///etc/containerpilot.json
```

I must not have pulled the latest image during my testing.